presto-hive/src/test/java/com/facebook/presto/hive/TestAriaHiveDistributedQueries.java

```diff
@@ -189,6 +191,15 @@ private Session ariaSession()
                 .build();
     }
 
+    Session noAriaSession()
+    {
+        return Session.builder(getQueryRunner().getDefaultSession())
+                .setSystemProperty(ARIA_SCAN, "false")
+                .setSystemProperty(ARIA_FLAGS, "0")
+            .setCatalogSessionProperty("hive", "aria_scan_enabled", "false")
```

Comment:
wrong indentation, HIVE_CATALOG, ARIA_SCAN_ENABLED

setCatalogSessionProperty(HIVE_CATALOG, ARIA_SCAN_ENABLED, "false")

(ARIA_SCAN_ENABLED field needs to be changed to public in com.facebook.presto.hive.HiveSessionProperties)